File: app/src/main/java/com/pioneer/aaron/dolly/IMainContract.java
package com.pioneer.aaron.dolly;

import android.content.Context;

/**
 * Created by Aaron on 4/18/17.
 */

public interface IMainContract {

    interface View extends IBaseView {

    }

    interface Presenter extends IBasePresenter {
        void forkRCS(Context context);

        void onDestroy(Context context);
    }

}

File: app/src/main/java/com/pioneer/aaron/dolly/IMainContract.kt
package com.pioneer.aaron.dolly

import android.content.Context

/**
 * Created by Aaron on 4/18/17.
 */

interface IMainContract {

    interface View : IBaseView

    interface Presenter : IBasePresenter {
        fun forkRCS(context: Context)

        override fun onDestroy(context: Context)

        fun vibrate()
    }

}
